.github/workflows/deploy-prod.yaml

```diff
@@ -0,0 +1,50 @@
+name: deploy-prod
+
+env:
+  DOCKERHUB_ORG: mobilecoin
+  REPO_NAME: block-explorer
+  TARGET_CLUSTER: utility-r1-d-k8s
+  CHART_REPO: https://harbor.mobilecoin.com/chartrepo/mobilecoinofficial-private
+  CHART_PATH: ./.internal-ci/charts/block-explorer
+  REACT_APP_RESERVE_AUDITOR_URL: https://auditor.mobilecoin.foundation/api
```

Comment:
remove these env vars:
REACT_APP_RESERVE_AUDITOR_URL
REACT_APP_FULL_SERVICE_URL

Reply:
Again we probably don't want to hard code this in the app.